db/migrate/20210208105018_clean_up_international_addresses_data.rb

```diff
@@ -1,26 +0,0 @@
-class CleanUpInternationalAddressesData < ActiveRecord::Migration[6.0]
```

Comment:
Deleted intentionally?

Reply:
Yeah - it's intentional (for now). That is the migration that stops the DB from being seeded. It's in a separate commit so we can easily revert it after a chat with the apply team.